### PR TITLE
WT-5400 Consistent wt verify debug output

### PR DESCRIPTION
**Pull request by jeremytay90**
* Added a function which prints time pairs in a consistent format

### CODE REVIEW

**Comment by luke-pearson** [comment]
Can you put an example of before / after output in the JIRA ticket, also still failing build here.

**Comment by jeremytay90** [comment]
Change of requirements - this will be done as part of WT-5345 but staying open until that is completed

File: src/btree/bt_debug.c [hunk 1]
```diff
@@ -1256,7 +1263,8 @@ __debug_cell(WT_DBG *ds, const WT_PAGE_HEADER *dsk, WT_CELL_UNPACK *unpack)
     WT_DECL_ITEM(buf);
     WT_DECL_RET;
     WT_SESSION_IMPL *session;
-    char ts_string[3][WT_TS_INT_STRING_SIZE];
+    char tp_string[2][WT_TP_STRING_SIZE];
+    char ts_string[WT_TS_INT_STRING_SIZE];
```
Comment (keithbostic) [inline src/btree/bt_debug.c]: This could be `char x, y;`.

File: src/btree/bt_vrfy.c [hunk 2]
```diff
@@ -375,18 +375,22 @@ __verify_tree(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK *addr_unpack
...
             addr = ref->addr;
             if (addr != NULL)
-                WT_RET(__wt_msg(session, "\t%s/%" PRIu64 ",%s/%" PRIu64,
-                  __wt_timestamp_to_string(addr->oldest_start_ts, ts_string),
-                  addr->oldest_start_txn, __wt_timestamp_to_string(addr->newest_stop_ts, ts_string),
-                  addr->newest_stop_txn));
+                WT_RET(__wt_msg(session, "\t%s,%s", __wt_time_pair_to_string(addr->oldest_start_ts,
+                                                      addr->oldest_start_txn, tp_string[0]),
+                  __wt_time_pair_to_string(
+                                  addr->newest_stop_ts, addr->newest_stop_txn, tp_string[1])));
```
Comment (keithbostic) [inline src/btree/bt_vrfy.c]: I'd personally be inclined to disallow multiple `-d debug` options, it doesn't make much sense to me and (as noted), the output will be corrupted, and this only fixes one case. I won't fight about it, though.

File: src/btree/bt_debug.c [hunk 3]
```diff
@@ -881,6 +884,10 @@ __debug_page_metadata(WT_DBG *ds, WT_REF *ref)
     if (mod != NULL)
         WT_RET(ds->f(ds, ", page-state=%" PRIu32, mod->page_state));
     WT_RET(ds->f(ds, ", memory-size %" WT_SIZET_FMT, page->memory_footprint));
+    if (addr != NULL)
+        WT_RET(ds->f(ds, ", start stop ts/txn %s,%s",
+          __wt_time_pair_to_string(addr->oldest_start_ts, addr->oldest_start_txn, tp_string[0]),
+          __wt_time_pair_to_string(addr->newest_stop_ts, addr->newest_stop_txn, tp_string[1])));
```
Comment (keithbostic) [inline src/btree/bt_debug.c]: Can't we get here with `WT_REF.addr` pointing onto the page's disk image?

I'm wondering if we should extend `__wt_ref_info()` to return the timestamp/txn information.

File: src/btree/bt_vrfy.c (hunk 2 above)
Comment (jeremytay90) [reply src/btree/bt_vrfy.c]: removed in https://github.com/wiredtiger/wiredtiger/pull/5138

File: src/btree/bt_debug.c (hunk 3 above)
Comment (jeremytay90) [reply src/btree/bt_debug.c]: done in https://github.com/wiredtiger/wiredtiger/pull/5138

File: src/btree/bt_debug.c (hunk 1 above)
Comment (jeremytay90) [reply src/btree/bt_debug.c]: There are other uses of this naming throughout the code. Shall I simplify them all?

File: src/btree/bt_debug.c (hunk 3 above)
Comment (keithbostic) [reply src/btree/bt_debug.c]: @luke-pearson, this change may be relevant to you.

**Pull request closed without merging**